lib/synced/strategies/full.rb

```diff
@@ -93,6 +90,10 @@ def perform
         end
       end
 
+      def reset_synced
+        # this is not supported by this strategy
```

Comment:
`NotImplementedError` sounds good. 
It may help with troubleshooting one day, just another check.